### DISCUSSION TITLE
为什么文字忽然变小，就无法识别了？需要怎么配置？

### DISCUSSION THREAD

**Discussion by Juan-hwt** [PaddlePaddle/PaddleOCR#13522] [Q&A]
![image](https://github.com/user-attachments/assets/18f714ed-6c36-49db-a1bb-29bb0805dcf7)

**Comment by UserWangZz** [comment]
可以具体一点吗？

**Comment by Juan-hwt** [comment]
我用pdocr提取表格数据，遇到上图中的那些忽然变小的字，提取出来是乱码。[源文件](https://www.tsrcc.cn/taishan/SMARTC/upload/file/20240423/1713863904907052422.pdf)
我感觉是，像素不够，而不是无法识别
所以问，如何调整像素（记得好像必须改成32的倍数，细节不清楚）
谢谢

**Reply by SWHL** [reply]
请提供相应提取代码和使用库的版本
